feat(content): Return to Paradise mission (adapted from #5112) (#9528)
endless-sky/endless-sky: data/human/human missions.txt

		dialog "Penny and Rosa thank you for getting them safely to <destination>, handing you <payment> for your troubles. You wish them and Eduardo the best of luck starting their new life away from a potential war in human space."
		log "Minor People" "Penny Little, Rosa Perkins, and Eduardo" `A young family who wanted to escape a potential war in human space. Took them to the planet Humanika, where the Quarg let humans settle.`

mission "Returning to Paradise"
	name "Take family home to <planet>"
	description "A family worried about the potential destruction in the Free Worlds wants to cut their vacation short and go home to <destination>."
	source
		government "Free Worlds"
		attributes "tourism"
	destination "Mainsail"

> 	passengers 3

Reviewer: Maybe the children share a bed?

> 	passengers 3

Author: I'll change it to four. It was 3 in the original PR.
```suggestion
	passengers 4
```

	cargo "personal belongings" 1
	to offer
		random < 30

> 		not "chosen sides"

Reviewer 2: ```suggestion
		not "chosen sides"
		has "assisted free worlds"
```

> 		not "Rescue Katya 1: accepted"

Reviewer 2: This condition doesn't exist. Checking for offered would be synonymous with checking if the mission were accepted (since you can't only either accept or defer the mission, and defer doesn't create any conditions).
```suggestion
		not "Rescue Katya 1: offered"
```

Although more generically, we could check if you've chosen sides.
```suggestion
		not "chosen sides"
```

			choice
				`	"Sure, I'll take you."`
				`	"Okay, but that's pretty far. This better be worth it."`
				`	"No, I don't think so. Sorry."`
					decline

			`	"Then it's a deal," she says as she gives you a firm handshake. The whole family gets onto the <ship> without a word, leaving you to load their many bags and cases.`
				accept

	on complete
		conversation
			branch luxury
				has "outfit: Luxury Accommodations"

			action
				payment 66999

> 			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, and every member of the family complained constantly about the lack of luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even told you their names. Although you wish you'd been paid more considering how terrible they were, you're happy to be rid of them.`

Reviewer 3: ```suggestion
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, and every family member complained constantly about the lack of luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leaves without a word of thanks; they never even tell you their names. You wish you'd been paid more considering how terrible they were, although you're happy to be rid of them.`
```

> 			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, and every member of the family complained constantly about the lack of luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even told you their names. Although you wish you'd been paid more considering how terrible they were, you're happy to be rid of them.`

Reviewer: > **`You wish`**

I don't think you should tell the player what they think. Maybe this?
```suggestion
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, and every member of the family complained constantly about the lack of luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even told you their names. You are finally rid of them, yet they only paid you a little compared to what you had to put up with.`
```

Author: It's not the first time that a like like "you're happy to be rid of them" appears... you also get "You are all too happy to return her to her owner" in the Pookie mission. I would think about changing it if it were a proper conversation, maybe? But, we might want to consider changing those instances later, too.

Reviewer: OK.

Author: I've changed it to try and remove that bit, how does it look now?

			action
				payment 66999
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, and every family member complained constantly about the lack of luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leaves without a word of thanks; they never even tell you their names. Considering what they put you through they should have paid you more, but nonetheless you're happy to be rid of them.`

> 				goto end

Reviewer 2: An endpoint at any point in an on complete conversation will end the conversation. No need for an empty label at the end of the conversation.
```suggestion
				decline
```

			label luxury
			action
				payment 120000
			`You're not sure if you've ever met a more pretentious and entitled family. The girls fought non-stop, but fortunately, the family was satisfied with the luxury accommodations on your ship. Once you land on <planet>, the lady gives you <payment> and the family leave without a word of thanks; they never even tell you their names. Nonetheless, you're happy to be rid of them.`

> 
> 			label end

Reviewer 2: ```suggestion
```
